### PR TITLE
Custom TMDB proxy and TMDB api key

### LINKED ISSUE #136: feat: option to host own tmdb-image proxy worker
Hi!

your proxy worker sometimes is rate limited and if you run `npm run build` it fails.

so i was considering to add an env variable to host the tmdb proxy .
let me know if it's doable since i haven't seen the source code for the proxy in your profile, if you open it i can consider implementing it myself!

### PR DESCRIPTION
**Pull request by GitGitro**
**What kind of change does this PR introduce?**
This pr introduces a new enviroment variable to add a custom GoodProxy url, closes #136 

**If relevant, did you update the documentation?**
Yes, updated the readme

**Summary**
With this even if the main proxy is rate limited, one can host his own GoodProxy worker.
Also solve the issue of running `npm run build` that fails if rate limited

**Other information**

**Edit**: renamed to TMDB proxy for better understanding

### CODE REVIEW

File: config/url.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const url = {
 };
 
 export const API_KEY = "171fe27dbfecc58e2a18fbced644cda9";
```
Comment (avalynndev) [inline config/url.ts:13]: can u fix this if possible too? ur code is perfect.. however this is another issue not related to this pr but if possible can u hide the API_key as well!?

**Review by avalynndev — CHANGES_REQUESTED** [review]
if you can please solve it or i will just merge the pr

File: config/url.ts (hunk 1 above)
Comment (GitGitro) [reply config/url.ts:13]: i was exxactly doing that too right now ahah
give me a few minutes and is ready

Comment (GitGitro) [reply config/url.ts]: done, hiding it entirely requires rewriting the git history and it's beyond the last commit so i'd say let's go with this

Comment (avalynndev) [reply config/url.ts]: Can you try deploying your fork to vercel and see if it works!?

Comment (GitGitro) [reply config/url.ts]: you're right i can't deploy on vercel, but i can build successfully locally, i'll investigate

Comment (avalynndev) [reply config/url.ts]: Well that's great 0-0 il try it out once if it works we are good to go

**Pull request merged**